reddit_liveupdate/templates/liveupdatemediaembedbody.html

```diff
@@ -45,7 +45,7 @@
         if ('MutationObserver' in window) {
             /* Currently just check on every set of mutations. */
             var observer = new MutationObserver(checkDimensions),
-                config = { attributes: true, subtree: true};
+                config = { attributes: true, subtree: true, childList: true};
```

Comment:
I was surprised too, it seems like `subtree` and `attributes` just checks if props of things change. I needed `childList` to make the observer fire for plain add/removes. Twitter embeds are an example of this, without `childList` the observer isn't fired when their placeholder blockquote is removed